quiche/src/lib.rs

```diff
@@ -7197,6 +7220,18 @@ impl<F: BufFactory> Connection<F> {
                     if ex_data.tx_cap_factor != self.tx_cap_factor {
                         self.tx_cap_factor = ex_data.tx_cap_factor;
                     }
+
+                    if let Some(discover) = ex_data.pmtud {
+                        let path = self.paths.get_active_mut()?;
+
+                        path.pmtud.enable(discover);
```

Comment:
I think that path mtu discovery needs to be done by path.  We're overriding here based on crypto handshake happening...  I assume that crypto handshake doesn't happen per path, how do we ensure that experiments that enable pmtud use it on every path, not just the first one created?

Reply:
Updated to enable pmtud on all existing paths for the connection